common/config.go

```diff
@@ -22,8 +22,8 @@ type Config struct {
 type Account struct {
 	Endpoint string
 	Region   string
-	Akid     string
-	Secret   string
+	Id       string
+	Token    string
```

Comment:
I dont think it belongs in the config since one spotinst account can have cross-cloud access.  I'm not sure we even need it for now, but I thought it might be useful (to switch-case it) in the handler or the library.

Reply:
ok, sounds good